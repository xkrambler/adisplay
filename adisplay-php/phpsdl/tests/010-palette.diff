003+ /usr/bin/php7.0: symbol lookup error: /data/dev/accord_display8th/adisplay/phpsdl/modules/sdl.so: undefined symbol: zend_objects_get_address
003- object(SDL_Palette)#%d (4) {
004-   ["ncolors"]=>
005-   int(1)
006-   ["version"]=>
007-   int(1)
008-   ["refcount"]=>
009-   int(1)
010-   ["colors"]=>
011-   array(1) {
012-     [0]=>
013-     object(SDL_Color)#%d (4) {
014-       ["r"]=>
015-       int(255)
016-       ["g"]=>
017-       int(255)
018-       ["b"]=>
019-       int(255)
020-       ["a"]=>
021-       int(255)
022-     }
023-   }
024- }
025- = Color
026- object(SDL_Color)#%d (4) {
027-   ["r"]=>
028-   int(255)
029-   ["g"]=>
030-   int(255)
031-   ["b"]=>
032-   int(255)
033-   ["a"]=>
034-   int(255)
035- }
036- 
037- %s: SDL_SetPaletteColors(): Invalid first color index, 2 >= 1 in %s/010-palette.php on line 9
038- int(-1)
039- int(0)
040- object(SDL_Color)#%d (4) {
041-   ["r"]=>
042-   int(1)
043-   ["g"]=>
044-   int(2)
045-   ["b"]=>
046-   int(3)
047-   ["a"]=>
048-   int(4)
049- }
050- = Free
051- = Done