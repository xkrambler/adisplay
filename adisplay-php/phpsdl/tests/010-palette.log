
---- EXPECTED OUTPUT
= Alloc
NULL
object(SDL_Palette)#%d (4) {
  ["ncolors"]=>
  int(1)
  ["version"]=>
  int(1)
  ["refcount"]=>
  int(1)
  ["colors"]=>
  array(1) {
    [0]=>
    object(SDL_Color)#%d (4) {
      ["r"]=>
      int(255)
      ["g"]=>
      int(255)
      ["b"]=>
      int(255)
      ["a"]=>
      int(255)
    }
  }
}
= Color
object(SDL_Color)#%d (4) {
  ["r"]=>
  int(255)
  ["g"]=>
  int(255)
  ["b"]=>
  int(255)
  ["a"]=>
  int(255)
}

%s: SDL_SetPaletteColors(): Invalid first color index, 2 >= 1 in %s/010-palette.php on line 9
int(-1)
int(0)
object(SDL_Color)#%d (4) {
  ["r"]=>
  int(1)
  ["g"]=>
  int(2)
  ["b"]=>
  int(3)
  ["a"]=>
  int(4)
}
= Free
= Done
---- ACTUAL OUTPUT
= Alloc
NULL
/usr/bin/php7.0: symbol lookup error: /data/dev/accord_display8th/adisplay/phpsdl/modules/sdl.so: undefined symbol: zend_objects_get_address
---- FAILED
