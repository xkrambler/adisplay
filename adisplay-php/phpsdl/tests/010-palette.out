= Alloc
NULL
/usr/bin/php7.0: symbol lookup error: /data/dev/accord_display8th/adisplay/phpsdl/modules/sdl.so: undefined symbol: zend_objects_get_address